cdap-ui/app/cdap/components/AbstractWidget/RadioGroupWidget/index.tsx

```diff
@@ -128,3 +128,10 @@ export default function RadioGroupWidget(props) {
 }
 
 (RadioGroupWidget as any).propTypes = WIDGET_PROPTYPES;
+(RadioGroupWidget as any).getWidgetAttributes = () => {
+  return {
+    layout: { type: 'string', required: true },
+    options: { type: 'IOption[]', required: true },
+    default: { type: 'string', required: false },
```

Comment:
where do we check for/use this?

Reply:
<img width="471" alt="Screen Shot 2020-07-30 at 2 07 41 PM" src="https://user-images.githubusercontent.com/14116152/88958136-09bf7800-d26e-11ea-9c26-66d79682930f.png">
This is used to set the default value of radio-group widget when we render it via `ConfigurationGroup` component.